modules/integration/tests/src/test/java/org/wso2/appmanager/integration/test/cases/admin/dashboard/businessowner/DeleteBusinessOwnerTestCase.java

```diff
@@ -0,0 +1,82 @@
+/*
+*Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*WSO2 Inc. licenses this file to you under the Apache License,
+*Version 2.0 (the "License"); you may not use this file except
+*in compliance with the License.
+*You may obtain a copy of the License at
+*
+*http://www.apache.org/licenses/LICENSE-2.0
+*
+*Unless required by applicable law or agreed to in writing,
+*software distributed under the License is distributed on an
+*"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+*KIND, either express or implied.  See the License for the
+*specific language governing permissions and limitations
+*under the License.
+*/
+package org.wso2.appmanager.integration.test.cases.admin.dashboard.businessowner;
+
+import org.json.JSONObject;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import org.wso2.appmanager.integration.utils.APPMAdminDashboardRestClient;
+import org.wso2.appmanager.integration.utils.AppmTestConstants;
+import org.wso2.appmanager.integration.utils.bean.BusinessOwner;
+import org.wso2.carbon.automation.engine.context.AutomationContext;
+import org.wso2.carbon.automation.engine.context.TestUserMode;
+import org.wso2.carbon.automation.engine.context.beans.User;
+import org.wso2.carbon.automation.test.utils.http.client.HttpResponse;
+
+/**
+ * Test case which verifies the ability of admin user of deleting business owner.
+ */
+public class DeleteBusinessOwnerTestCase {
+    private static final String TEST_DESCRIPTION = "Verify deleting business owner.";
+    private APPMAdminDashboardRestClient appmAdminDashboardRestClient;
+    private String businessOwnerName = "DeleteBusinessOwnerTestCase_";
+    private static AutomationContext appMServerSuperTenant;
+
+    @BeforeClass(alwaysRun = true)
+    public void startUp() throws Exception {
+        appMServerSuperTenant = new AutomationContext(AppmTestConstants.APP_MANAGER, TestUserMode.SUPER_TENANT_ADMIN);
+    }
+
+    @Test(dataProvider = "validUserModeDataProvider", description = TEST_DESCRIPTION)
+    public void testDeleteBusinessOwnerWithValidUsers(String userName, String password, AutomationContext
+            automationContext) throws Exception {
+        String backEndUrl = automationContext.getContextUrls().getWebAppURLHttps();
+        appmAdminDashboardRestClient = new APPMAdminDashboardRestClient(backEndUrl);
+        appmAdminDashboardRestClient.login(userName, password);
+
+        int businessOwnerId = createBusinessOwner(appmAdminDashboardRestClient, userName);
+        HttpResponse deleteBusinessOwnerResponse =  appmAdminDashboardRestClient.deleteBusinessOwner(businessOwnerId);
+        // logout from admin dashboard rest client.
```

Comment:
Fixed in https://github.com/wso2/product-app-manager/commit/e263afed698e7dbd1253c0136563513fd4051eea